### ISSUE TITLE
A lot of rules are failing due to HTTPS certificate expirations

### ISSUE THREAD

**Issue by geek1011** [just-install/just-install-updater-go#11]
Many rules are failing due to the certificates expiring (about 4 expired today so far) ... I can either temporarily disable the certificate check we could just wait.

cc @lvillani @cyagon

**Comment by lvillani** [comment]
Can you skip rules that are failing (and print a warning)? I guess most site owners are going to fix their certificates, I'm hopeful that this is going to resolve by itself over time, and I think it's safest to skip their host until they sort it out.

**Comment by geek1011** [comment]
@lvillani Yes, I can skip rules (I implemented that a week ago). I already skip the virtualbox rule because the latest version is not available for Windows yet.

**Comment by geek1011** [comment]
The build passes now: https://github.com/just-install/registry/commit/c9f77a7f83d5fa3acf5839e501d76ddd69dcc721
